Automattic/WP-Job-Manager
# Fixed check for existing tinymce description field in job-submission.js

Looks like the operator was incorrect in the `descriptionFieldIsPresent()` function, it should use an `&&` to only pass if `tinymce` var exists AND we have a `job_description` field.

I also ran the `gulp` command to minify the job-submission.js file. See #1856.

**To Test**
* Verify that an empty job description field that is using tinymce shows a validation error.
* Change the field type from `wp-editor` to `text` [at this line](https://github.com/Automattic/WP-Job-Manager/blob/master/includes/forms/class-wp-job-manager-form-submit-job.php#L240), and verify that it also shows a verification message with no errors in the js console.
